modules/local/python/igv_session.nf

```diff
@@ -9,12 +9,15 @@ process IGV_SESSION {
 
     input:
     path genome
-    path gtf
+    path genome_index
+    //path gtf
```

Comment:
gtf can be removed?

Reply:
yes but i may switch back to gtf in the future so I'm keeping it in for now